textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -32,78 +32,86 @@
 import edu.uci.ics.textdb.common.field.TextField;
 
 public class Utils {
-    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException{
+    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException {

```

Comment:
Why does github show these differences?  Just because of reformatting? If so, it will be really good to separate reformatting changes and logic changes.